xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
    </itemizedlist>
   </sect2>
 
+  <sect2 xml:id="sec.depl.maintenance.postgre-mariadb.upgrade">
+   <title>Preparing &postgres;-Based &productname; 7 for Upgrade</title>
+   <para>
+    Upgrading &productname; 7 is only possible when it uses &mariadb; deployed
+    with the Database &barcl;. This means that before you can proceed with
+    upgrading &productname; 7, you must migrate &postgres; to &mariadb; first. The following description covers several possible scenarios.
+   </para>
+   <sect3 xml:id="sec.postgre-mariadb.upgrade.scenario1">
+    <title>Non-HA setup or HA Setup with a 2-Node Cluster and
+    &postgres; Database Backend</title>
+    <para>
+     Before you proceed, make sure that the database cluster has an odd number
+     of nodes. If necessary, extend the cluster as described in <xref linkend="sec.postgre-mariadb.upgrade.scenario2"/>. Install the latest maintenance updates on &productname; 7. In the
+     &crow; &wi;, switch to the <guimenu>Database</guimenu> &barcl;. You should
+     see the new <systemitem>mysql-server</systemitem> role in the
+     <guimenu>Deployment</guimenu> section. Add your &dbnode; to the
+     <systemitem>mysql-server</systemitem> role and apply the
+     &barcl;. &mariadb; is now deployed and running, but it is still not used
+     as a back end for &ostack; services. Do not change the
+     <literal>sql_engine</literal> at this point.
+    </para>
+    <!-- FIXME: Currently doesn't work accoring to devs
+	 <para>
+     If &rabbit; is using DRBD, enable the <guimenu>Use RabbitMQ
+     Clustering</guimenu> option in the &rabbit; &barcl; under <guimenu>High Availability</guimenu>. Since DRBD is not available
+     in &productname; 8, and &rabbit; clustering is supported in &productname;
+     7, you should switch to this option now.
+    </para> -->
+    <para>
+     Follow <xref linkend="postgre.mariadb.data.migration"/> to migrate the data from &postgres; to &mariadb;.
+    </para>
+    <procedure xml:id="postgre.mariadb.data.migration">
+     <title>Data Migration</title>
+     <step>
+      <para>
+           Run the <systemitem>/opt/dell/bin/prepare-mariadb</systemitem>
+	   script on the &admnode; to prepare the &mariadb; instance by creating
+	   the required users, databases, and tables.
+      </para>
+     </step>
+     <step>
+      <para>
+       After the script is finished, you'll find a list of all databases and
+       URLs that are ready for data migration in the
+       <filename>/etc/pg2mysql/databases.yaml</filename>. The script's output
```

Comment:
The yaml file will be located "at one of the database nodes."

Reply:
I dropped this, because the text on line 568 explains that the Summary of used databases line shows the exact location of the /etc/pg2mysql/databases.yaml file.

Reply:
Fixed.